sdk/communication/AzureCommunicationChat/Source/ChatThreadClient.swift

```diff
@@ -317,21 +317,19 @@ public class ChatThreadClient {
 
     /// Updates a message.
     /// - Parameters:
-    ///    - content: The updated message content.
+    ///    - message: The updated message.
     ///    - messageId: The message id.
     ///    - options: Update chat message options
     ///    - completionHandler: A completion handler that receives a status code on success.
     public func update(
-        content: String,
+        message: UpdateChatMessageRequest,
```

Comment:
It seems content and message are now used inconsistently. Before I think it was always "content". Now it's content half the time and message half the time.

Reply:
The parameter is always "message" now, before we had:
`send(message: SendChatMessageRequest)`
`update(content: String)`

Mainly because you could only update the content anyways so the method would construct the UpdateChatMessageRequest internally.

Now we have:
`send(message: SendChatMessageRequest)`
`update(message: UpdateChatMessageRequest)`